monitor/monitor.go

```diff
@@ -301,41 +158,78 @@ func RunMonitor(ctx context.Context, curRef string, options *controllerapi.Build
 	}
 }
 
+func printHelpMessage(out io.Writer, registeredCommands map[string]types.Command) {
+	var names []string
+	for name := range registeredCommands {
+		names = append(names, name)
+	}
+	sort.Strings(names)
+	fmt.Fprint(out, "Available commands are:\n\n")
```

Comment:
```suggestion
	fmt.Fprint(out, "Available commands are:\n")
```
Let's keep the same format as before.